wolfcrypt/src/ecc.c

```diff
@@ -6884,6 +6900,8 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
    CRYS_ECPKI_HASH_OpMode_t hash_mode;
 #elif defined(WOLFSSL_SILABS_SE_ACCEL)
    byte sigRS[ECC_MAX_CRYPTO_HW_SIZE * 2];
+#elif defined(WOLFSSL_SE050)
+   byte sigRS[256]; //ECC_MAX_CRYPTO_HW_SIZE *2 results in 512, will just have to run and see if that size is too large
```

Comment:
Physical SE050 hardware limit is 521 bits (66 bytes * 2 or 132). This comment is wrong. Just use `ECC_MAXSIZE * 2`.